File: Matnas/Common/VolunteerAndEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
    public class VolunteerAndEvent
    {
        public int Id { get; set; }
        public Event Event { get; set; }
        public Volunteer Volunteer { get; set; }
        public Category Category { get; set; }
        public string Comments { get; set; }
        public Nullable<System.DateTime> DateAdded { get; set; }
    }
}

File: rmatnas/src/app/Classes/VolunteerAndEvent.ts
import { Eventt } from './Eventt';
import { Volunteer } from './Volunteer';
import { Category } from './Category';

export class VolunteerAndEvent {
    Id: number;
    Event: Eventt;
    Volunteer: Volunteer;
    Category: Category;
    Comments: string;
    DateAdded: string;

    constructor(event: Eventt, volunteer: Volunteer, category: Category, comments: string, dateAdded: string) {
        this.Comments = comments;
        this.Category = category;
        this.Event = event;
        this.Volunteer = volunteer;
        this.DateAdded = dateAdded;

    }
}
